### PR TITLE
Fix bug with successful event table and successful interevent table

### PR DESCRIPTION
**Pull request by MartinMDBA**
Fix bug that was filtering out spells of exactly the required length from the successful events table and the flow on impact to the successful interevent table. Update tests.

### CODE REVIEW

File: py_ewr/scenario_handling.py [hunk 1]
```diff
@@ -36,29 +36,29 @@
     
 #     return df
 
-# def standard_time_series_row_filler(input_df: pd.DataFrame, date_range: pd.PeriodIndex) -> pd.DataFrame:
-#     ''' Inserts blank rows in standard time series formatted data frames 
-#     where data for dates are missing to allow for analysis of input files with gaps in daily data
+def standard_time_series_row_filler(input_df: pd.DataFrame, date_range: pd.PeriodIndex) -> pd.DataFrame:
```
Comment (ElishaMDBA) [inline py_ewr/scenario_handling.py:39]: delete this function

File: py_ewr/summarise_results.py [hunk 2]
```diff
@@ -569,20 +569,22 @@ def filter_successful_events(all_events: pd.DataFrame, ewr_table_path: str = Non
...
+        ewr = i.split('TEMPORARY_ID_SPLIT')[3]      
+        # print(df_subset) 
 
         # Pull EWR minSpell value from EWR dataset
         minSpell = int(data_inputs.ewr_parameter_grabber(EWR_table, gauge, pu, ewr, 'MinSpell'))
         # Filter out the events that fall under the minimum spell length
-        df_subset = df_subset.drop(df_subset[df_subset.eventDuration <= minSpell].index)
-
+        df_subset = df_subset.drop(df_subset[df_subset.eventDuration < minSpell].index)
+        # print(df_subset)
```
Comment (ElishaMDBA) [inline py_ewr/summarise_results.py:582]: delete print statement

File: py_ewr/summarise_results.py [hunk 3]
```diff
@@ -569,20 +569,22 @@ def filter_successful_events(all_events: pd.DataFrame, ewr_table_path: str = Non
...
         # Append to master dataframe
         all_successfulEvents = pd.concat([all_successfulEvents, df_subset], ignore_index=True)
+        # print(all_successfulEvents)
```
Comment (ElishaMDBA) [inline py_ewr/summarise_results.py:585]: delete print statement

File: py_ewr/summarise_results.py [hunk 4]
```diff
@@ -569,20 +569,22 @@ def filter_successful_events(all_events: pd.DataFrame, ewr_table_path: str = Non
...
     all_successfulEvents.drop(['ID', 'multigaugeID'], axis=1, inplace=True)
-
+    # print(all_successfulEvents)
```
Comment (ElishaMDBA) [inline py_ewr/summarise_results.py:587]: delete print statement

File: py_ewr/scenario_handling.py [hunk 5]
```diff
@@ -495,6 +494,10 @@ def process_scenarios(self):
             elif self.model_format == 'ten thousand year':
                 df = pd.read_csv(scenarios[scenario], index_col = 'Date')
                 df_F, df_L = cleaner_ten_thousand_year(df, self.parameter_sheet)
+
+            elif self.model_format == 'Source - Qld':
```
Comment (ElishaMDBA) [inline py_ewr/scenario_handling.py:498]: delete lines 497 - 500

File: tests/test_scenario_handling.py [hunk 6]
```diff
@@ -94,35 +95,58 @@ def test_extract_gauge_from_string():
...
-    str_df = str_df.set_index('Date')
-    df_f, df_l = scenario_handling.cleaner_standard_timeseries(str_df)
-    
+    input_df = input_df.set_index('Date')
+    df_f, df_l = scenario_handling.cleaner_standard_timeseries(input_df)
+
     # Set up expected data and test:
     expected_df_flow = input_df.copy(deep=True)
-    expected_df_flow = expected_df_flow.set_index('Date')
+    # expected_df_flow = expected_df_flow.set_index('Date')
```
Comment (ElishaMDBA) [inline tests/test_scenario_handling.py:114]: delete commented out code

**Review by ElishaMDBA — CHANGES_REQUESTED** [review]

**Comment by MartinMDBA** [comment]
Will resubmit request to address comments

**Pull request closed without merging**